src/loop_thread.hpp

```diff
@@ -26,7 +26,6 @@ class LoopThread {
 public:
   LoopThread()
       : loop_(uv_loop_new())
```

Comment:
It  looks like we're still using the deprecated loop functions here. Did you intend to change that for libuv 1.x to address [CPP-194](https://datastax-oss.atlassian.net/browse/CPP-194)?